internal/hud/renderer.go

```diff
@@ -181,12 +179,22 @@ func (r *Renderer) renderStatusBar(v view.View) rty.Component {
 			errorCountMessage = fmt.Sprintf(" %d %s", errorCount, s)
 		}
 
-		if v.TiltfileErrorMessage() != "" {
-			_, _ = tiltfileError.WriteString(" • Tiltfile error")
-		}
-		sb.Fg(cBad).Text("✖").Fg(tcell.ColorDefault).Fg(cText).Textf("%s%s", errorCountMessage, tiltfileError.String()).Fg(tcell.ColorDefault)
+		sb.Fg(cBad).Text("✖").
+			Fg(cText).Textf("%s", errorCountMessage)
 	}
-	return rty.Bg(rty.OneLine(sb.Build()), tcell.ColorWhiteSmoke)
+	return sb.Build()
+}
+
+func (r *Renderer) renderStatusBar(v view.View) rty.Component {
+	l := rty.NewConcatLayout(rty.DirHor)
+	l.Add(rty.TextString(" "))
+	l.Add(r.renderStatusMessage(v))
+	l.Add(rty.TextString(" "))
+	l.AddDynamic(rty.NewFillerString(' '))
+
+	msg := " Terminal is limited. For more, open Tilt Web (enter) → "
```

Comment:
```suggestion
	msg := " Open Tilt in browser (enter) for full experience. Terminal is limited "	
```

@nicks : I don't think `Tilt Web` as a term is used anywhere and don't necessarily think we should use it, since it could confuse with Tilt Cloud. I think we should start with a positive call to action, before saying Terminal is limited. This is similar to how when you visit a web app with your phone, often there's a banner that says, "Download the iOS app for the full experience". So a positive reason (full experience) to use web instead of (or before) a negative reason (Terminal is limited).

Maybe then we don't even have to say "Terminal is limited", then?

Reply:
i like the idea of focusing on the positive. unfortunately, that text is too long

Reply:
@nicks : I like your latest version since it assumes the browser is the default way to "explore".